frontend/src/lib/lemon-ui/LemonCard/LemonCard.tsx

```diff
@@ -25,6 +33,19 @@ export function LemonCard({
             onClick={onClick}
```

Comment:
commenting here cos you can't comment on unchanged lines
we should have a different cursor when there's an onclick handler so folk can see it is clickable